android/src/main/java/com/amplitude/android/Amplitude.kt

```diff
@@ -112,6 +112,17 @@ open class Amplitude(
         }
     }
 
+    internal fun setDeviceId(deviceId: String, expectedCurrentDeviceId: String?): Amplitude {
```

Comment:
Can this logic be done in `core/Amplitude` instead to fix all platforms?

Reply:
Removed now. Was added to `android/Amplitude` to be `internal`